Userland/Libraries/LibWasm/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ set(SOURCES
 )
 
 serenity_lib(LibWasm wasm)
-target_link_libraries(LibWasm PRIVATE LibCore)
+target_link_libraries(LibWasm PRIVATE LibCore LibJS)
```

Comment:
Seems kinda wrong to connect LibWasm to LibJS outside of the LibWeb glue code, but I assume you know what you're doing so...

Reply:
Not much else I can do, I could skip the libjs link if I skip using Value's methods, but that's only really gonna work until someone decides to move something in `JS::Completion` out of line; so might as well just link it and call it good.

Reply:
That is, unless you want to do weird stuff like move the JS bindings into a separate library, or into LibJS